relayer-cli/src/commands/tx/channel.rs

```diff
@@ -152,53 +153,6 @@ pub struct TxRawChanOpenTryCmd {
 
 impl Runnable for TxRawChanOpenTryCmd {
     fn run(&self) {
-        let config = app_config();
```

Comment:
Deleted this lines because it was a coding error. It's enough to call into `tx_chan_cmd!` macro, we don't need these manual steps since the macro handles everything.